plugins/catalog-backend/config.d.ts

```diff
@@ -334,6 +334,19 @@ export interface Config {
         }>;
       };
 
+      /**
+       * AwsOrganizationCloudAccountProcessor configuration
+       */
+      awsOrganization?: {
+        providers: Array<{
```

Comment:
You'll see that providers generally have a key that says _which_ provider that it applies to. Should generally be `host: string` as a hostname, as used in the `integrations` config. The ones in this file are old and use a `target` URL, but we're probably moving away from that.

You aren't going to want to have all ARNs apply to every target system right? Or is it actually the case that there will only ever be one single ARN and therefore you would want to just remove the `providers` array and make it a single key instead?

Reply:
@freben the idea was that eventually, I think this processor should support multiple providers. It's an edge case, but I wouldn't say very rare for an organization to perhaps have multiple organizations that they want to ingest (say as a result of acquisitions, migrations etc). 

While this support is not there today, it felt better to make this change in anticipation of that, but i'm fine going back to a single configuration as well and adding multiple later.

Reply:
Yeah, actually going to make this simpler and design for now. It's easy to change later to support multiple.